src/client/common/application/encryptedStorage.ts

```diff
@@ -53,34 +22,20 @@ export class EncryptedStorage implements IEncryptedStorage {
             this.testingState.set(`${service}#${key}`, value || '');
             return;
         }
-        // When not in insiders, use keytar
-        if (!this.useNativeNb) {
-            if (!value) {
-                await keytar?.deletePassword(service, key);
-            } else {
-                return keytar?.setPassword(service, key, value);
-            }
+
+        if (!value) {
+            await this.extensionContext.secrets.delete(`${service}.${key}`);
         } else {
-            if (!value) {
-                await this.authenService.deletePassword(`${service}.${key}`);
-            } else {
-                await this.authenService.setPassword(`${service}.${key}`, value);
-            }
+            await this.extensionContext.secrets.store(`${service}.${key}`, value);
         }
     }
     public async retrieve(service: string, key: string): Promise<string | undefined> {
         // On CI we don't need to use keytar for testing (else it hangs).
         if (IS_REMOTE_NATIVE_TEST && this.extensionContext.extensionMode !== ExtensionMode.Production) {
```

Comment:
Do we still need this anymore without keytar?

Reply:
Good question, I'll create an issue to check that, right now tests won't run at all because of the busted nature of the notebook test due to the API issues.